cpp/tests/mtmg/threaded_test.cu

```diff
@@ -107,7 +107,7 @@ class Tests_Multithreaded
     ncclGetUniqueId(&instance_manager_id);
 
     auto instance_manager = resource_manager.create_instance_manager(
-      resource_manager.registered_ranks(), instance_manager_id);
+      resource_manager.registered_ranks(), instance_manager_id, 4);
```

Comment:
What is 4 here?

Reply:
Made this a constant, added a comment describing why it's 4 in the latest push.